packages/frontend/src/components/common/ResourceView/ResourceViewGrid/ResourceViewGridSpaceItem.tsx

```diff
@@ -36,107 +54,119 @@ const ResourceViewGridSpaceItem: FC<ResourceViewGridSpaceItemProps> = ({
 
     const theme = useMantineTheme();
 
-    return (
-        <Paper
-            ref={ref}
-            p={0}
-            withBorder
-            bg={hovered ? theme.fn.rgba(theme.colors.gray[0], 0.5) : undefined}
-            h="100%"
-        >
-            <Group p="md" align="center" spacing="md" noWrap>
-                <ResourceIcon item={item} />
-
-                <Stack spacing={4} sx={{ flexGrow: 1, flexShrink: 1 }}>
-                    <Text lineClamp={1} fz="sm" fw={600}>
-                        {item.data.name}
-                    </Text>
+    const renderAccess = useCallback(() => {
```

Comment:
would be nice to have a enum
```
enum ResourceAccess {
    Private = 'private',
    Public = 'public',
    Shared = 'shared',
}
```

also, have a function with signature `getResourceAccessType(item: ResourceItem): ResourceAccess`
which will be used here with switch/case statement to determine what to render. you can also either extract whatever gets rendered as a separate component like you did for `AttributeCount`.